client/src/css/login.css

```diff
@@ -0,0 +1,57 @@
+#login-div{
+    margin-top:9%;
+    left:10%;
+    padding: 7% 0 0 0;
+    background: -webkit-linear-gradient(#204d74, #a6e1ec); /* For Safari 5.1 to 6.0 */
+    background: -o-linear-gradient(#204d74, #a6e1ec); /* For Opera 11.1 to 12.0 */
+    background: -moz-linear-gradient(#204d74, #a6e1ec); /* For Firefox 3.6 to 15 */
+    background: linear-gradient(#204d74, #a6e1ec); /* Standard syntax */
+    border: 2px solid black;
+    border-radius: 5px;
+    width:60%;
+    height:520px;
+    opacity: 0.9;
+    filter: alpha(opacity=60);
```

Comment:
IE hack? Not needed on modern browsers. Which browsers are being supported here?